pkg/agent/monitortool/monitor.go

```diff
@@ -206,6 +209,12 @@ func (m *NodeLatencyMonitor) onNodeLatencyMonitorUpdate(oldObj, newObj interface
 func (m *NodeLatencyMonitor) updateLatencyConfig(nlm *v1alpha1.NodeLatencyMonitor) {
 	pingInterval := time.Duration(nlm.Spec.PingIntervalSeconds) * time.Second
 
+	if pingInterval < MinReportIntervalSeconds*time.Second {
+		m.reportInterval = MinReportIntervalSeconds * time.Second
+	} else {
+		m.reportInterval = pingInterval
```

Comment:
maybe as a "cheap" alternative to jitter, we could set this to `m.reportInterval = pingInterval + 1*time.Second` instead? This way we could ensure that the 2 tickers don't tick in lockstep.